deploy/azure/install-agent.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euxo pipefail
+
+usage="$0 <elastic agent version> <elastic artifact server> <fleet url> <enrollment token>"
+ElasticAgentVersion=${1:?$usage}
+ElasticArtifactServer=${2:?$usage}
+FleetUrl=${3:?$usage}
+EnrollmentToken=${4:?$usage}
+
+ElasticAgentArtifact="elastic-agent-$ElasticAgentVersion-linux-x86_64"
```

Comment:
I went with x86 here, should we use arm for azure as well for consistency?

Reply:
Most of our test if not all of them are on Linux x86 machines AFAIK, so I think it's better to have it like that